Epsilon.Abstractions/OutcomeGradeStatus.cs

```diff
@@ -1,5 +1,9 @@
+using System.Diagnostics.CodeAnalysis;
+
 namespace Epsilon.Abstractions;
 
+//Suppressed for later implementation. Remove when implemented  
+[SuppressMessage("ReSharper", "UnusedType.Global")]
```

Comment:
You can also do it like this, which is the most common way: `[SuppressMessage("a", "b", Justification = "justify here")]`